zap/src/test/resources/org/zaproxy/zap/spider/parser/html/AppletElementsSpiderHtmlParser.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+<head>
+<base href="http://example.com/sample/">
+<meta charset="UTF-8">
+<title>Input Elements - Spider HTML Parser</title>
+</head>
+<body>
+<!--  Ignored: -->
+<applet></applet>
+
+<!--  src -->
+<applet type="deprecated" src="/cool/applet/file.class"  alt="badpractice" code="somecode" height="320" width="480" >
+</applet>
+
+<applet code="game.class" align="left" archive="/game.zip" height="250" width="350">
+    <param name="difficulty" value="easy">
+    <b>Sorry, you need Java to play this game.</b>
+</applet>
+
+<applet code="upgradedGame.class" align="left" src="https://example.com/game/applet/file.class" height="250" width="350">
+    <param name="difficulty" value="easy">
+    <b>Sorry, you need Java to play this upgraded game.</b>
+</applet>
+
+<applet code="codebase.class" align="left" codebase="https://example.com/absolute/applet/" height="250" width="350">
+</applet>
+
+<applet code="codebase.class" align="left" codebase="/relative/applet/" height="250" width="350">
```

Comment:
With the leading slash this is absolute right? (Ex: from server root this is the full actual path.)

Reply:
You are "absolutely" right :)